storage/postgreskv/client.go

```diff
@@ -89,12 +89,14 @@ func (client *Client) GetPath(ctx context.Context, bucket, key storage.Key) (_ s
 	row := client.pgConn.QueryRow(q, []byte(bucket), []byte(key))
 	var val []byte
 	err = row.Scan(&val)
-	if err == sql.ErrNoRows {
-		return nil, storage.ErrKeyNotFound.New(key.String())
-	}
 	if err != nil {
```

Comment:
Nit: Again, this check is probably just extra code / against Go style guidelines.